--- linux-4.5.2-CACS/fs/fcntl.c	2016-05-04 11:24:07.028116948 -0500
+++ linux-4.5.2/fs/fcntl.c	2016-04-20 01:46:35.000000000 -0500
@@ -27,7 +27,7 @@
 #include <asm/siginfo.h>
 #include <asm/uaccess.h>
 
-#define SETFL_MASK (O_APPEND | O_NONBLOCK | O_NDELAY | O_DIRECT | O_NOATIME )
+#define SETFL_MASK (O_APPEND | O_NONBLOCK | O_NDELAY | O_DIRECT | O_NOATIME)
 
 static int setfl(int fd, struct file * filp, unsigned long arg)
 {
@@ -736,17 +736,12 @@ EXPORT_SYMBOL(kill_fasync);
 
 static int __init fcntl_init(void)
 {
-  /**
-    @author ceneblock
-    @date 2016-05-04
-    @brief adds in the O_CLASSIFY for CACS_improvements
-  */
 	/*
 	 * Please add new bits here to ensure allocation uniqueness.
 	 * Exceptions: O_NONBLOCK is a two bit define on parisc; O_NDELAY
 	 * is defined as O_NONBLOCK on some platforms and not on others.
 	 */
-	BUILD_BUG_ON(22 - 1 /* for O_RDONLY being 0 */ != HWEIGHT32(
+	BUILD_BUG_ON(21 - 1 /* for O_RDONLY being 0 */ != HWEIGHT32(
 		O_RDONLY	| O_WRONLY	| O_RDWR	|
 		O_CREAT		| O_EXCL	| O_NOCTTY	|
 		O_TRUNC		| O_APPEND	| /* O_NONBLOCK	| */
@@ -754,7 +749,7 @@ static int __init fcntl_init(void)
 		O_DIRECT	| O_LARGEFILE	| O_DIRECTORY	|
 		O_NOFOLLOW	| O_NOATIME	| O_CLOEXEC	|
 		__FMODE_EXEC	| O_PATH	| __O_TMPFILE	|
-		__FMODE_NONOTIFY | O_CLASSIFY
+		__FMODE_NONOTIFY
 		));
 
 	fasync_cache = kmem_cache_create("fasync_cache",
--- linux-4.5.2-CACS/include/uapi/asm-generic/fcntl.h	2016-05-04 11:15:27.959411797 -0500
+++ linux-4.5.2/include/uapi/asm-generic/fcntl.h	2016-04-20 01:46:35.000000000 -0500
@@ -88,15 +88,6 @@
 #define __O_TMPFILE	020000000
 #endif
 
-/**
-  @author ceneblock
-  @date 2016-05-04
-  @brief adds in the O_CLASSIFY for CACS_improvements
-*/
-#ifndef O_CLASSIFY
-#define O_CLASSIFY 040000000
-#endif
-
 /* a horrid kludge trying to make sure that this will fail on old kernels */
 #define O_TMPFILE (__O_TMPFILE | O_DIRECTORY)
 #define O_TMPFILE_MASK (__O_TMPFILE | O_DIRECTORY | O_CREAT)      
